install.sh

```diff
@@ -98,7 +98,9 @@ mkdir -p $STATIC_PATH
 cp $SOURCE_PATH/files/rw_test_audio1.wav $MEDIA_PATH
 
 # Copy default production settings file
+mkdir $WWW_PATH/settings
```

Comment:
We need to the `-p` in the mkdir command so that running install.sh a second time will not error.